src/components/ModalConfirmation/ModalConfirmation.tsx

```diff
@@ -21,10 +25,17 @@ const ModalConfirmation = (props: Props) => {
       case 'buy_meal':
         return `We appreciate your support for ${props.sellerName} and for those in need! Please check your email for your receipt.`;
       default:
```

Comment:
do we need a default case?

Reply:
if this is an unexpected case, should we log here?